packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts

```diff
@@ -288,7 +299,7 @@ export class HorizontalBarChart extends FASTElement {
       .append('div')
       .append('span')
       .attr('class', 'chart-title')
-      .text(data?.chartTitle ? data?.chartTitle : '');
+      .text(data?.barTitle ? data?.barTitle : '');
```

Comment:
Is the title for each row or all rows together.

Reply:
title for each row

Reply:
lets call it chartSeriesTitle

Reply:
done